README.adoc

```diff
@@ -1,20 +1,18 @@
 = HackADoc
 
 This is a fork of the great http://hackmd.io[hackmd.io].
-We just thought it would be even greater with asciidoctor.js ;)
-
-So we integrated https://github.com/asciidoctor/asciidoctor.js[asciidoctor.js] and changed the name accordingly.
+We just thought it would be even greater with AsciiDoctor and so we included https://github.com/asciidoctor/asciidoctor.js[Asciidoctor.js] into the project and changed its name accordingly ;)
 
 == Status
 
-Using AsciiDoc.js, the base functionality, typing AsciiDoc and seeing a nice rendered document is working fine.
-Also, uploading and embedding images is working.
+Thanks to Asciidoc.js, the base functionality (typing AsciiDoc and seeing a nice rendered document) workis fine.
```

Comment:
> the base functionality (typing AsciiDoc and seeing a nice rendered document) **workis fine.**

typo?

Reply:
Yepp :grin: Fixed it with 759249ef452fded76975371daf74659864e9ff93